client/packages/system/src/Program/api/operations.graphql

```diff
@@ -0,0 +1,30 @@
+fragment ProgramDocument on DocumentRegistryNode {
```

Comment:
I was really hoping to have a smaller fragment, just the props that we need for the list.. and then for the creation, which requires the entire documentRegistry node.. so everything is in.
Except the infinite recursion of the `children`, which I'm hoping isn't necessary!